src/player.cpp

```diff
@@ -772,10 +772,11 @@ void player::process_turn()
     // Didn't just pick something up
     last_item = itype_id( "null" );
 
-    if( has_active_bionic( bio_metabolics ) && power_level + 25 <= max_power_level &&
-        0.8f < get_kcal_percent() && calendar::once_every( 5_turns ) ) {
-        mod_stored_kcal( -25 );
-        charge_power( 25 );
+    if( has_active_bionic( bio_metabolics ) && power_level < max_power_level &&
+        0.8f < get_kcal_percent() && calendar::once_every( 1_turns ) ) {
+        // Efficiency is approximately 25%, power output is ~60W
```

Comment:
60W * 6 seconds = 360 J = 1/3rd of battery charge.  Should be -3 kcals every 3 turns for 1 power.

Reply:
Did I math wrong?  1 kcal = 4.184 kjoule, meaning 1:1 is approximately 25% efficient.

I definitely did get the power output wrong as 1kJ / 6 seconds is 600kJ/hr which is ~167 Wh, so 1 every 3 turns is about right (a little high, no big deal).

Reply:
okay, you got the efficiency right but the rate wrong.

Reply:
Hooray for stating intentions, thanks for the correction.